resources/guidelines/testing/store/quality-guidelines-plugins/README.md

```diff
@@ -244,6 +246,29 @@ A test button for optional API access data must be available. If the API data is
 
 [Example for implementing an API Test Button into the System Config form](https://github.com/shyim/ShyimApiTest)
 
+### External technology apps / STP apps
+
+Every external technology app needs to track their commission. This is an example to implement the tracking 
```

Comment:
This information is unclear.

Reply:
Which information do you think is missing?